### PR TITLE
bad code

### PR DESCRIPTION
**Pull request by Legoburner73**
hi this is bad code hardly works, bye

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//import { setInterval } from "core-js/library/web/timers";
```
Comment (haykam821) [inline app.js:1]: What?

File: app.js [hunk 2]
```diff
@@ -489,24 +660,25 @@ class Player extends Entity {
 
 		this.hasUpgraded = false;
 		this.canUpgrade = true;
-		this.dev = false;
+		this.dev = this.name === "5dfh7s4GFD5" ? true : false;
```
Comment (haykam821) [inline app.js:663]: What is this for?

File: app.js [hunk 3]
```diff
@@ -625,14 +801,17 @@ class Player extends Entity {
...
 					parent: this.id,
 					angle: angle,
+					barrel: classes[classes[this.tank].barrels[0].bulletType].barrels,
+					tank: classes[this.tank].barrels[0].bulletType,
+					parent_tank: this.tank
 				});
 				b.x = this.x - 10;
 				b.y = this.y;
-				if (this.tank === "bomber" || this.tank === "grenadier") {
+				if (this.tank === "bomber" || this.tank === "grenadier" || this.tank === "rocketeer") {
```
Comment (haykam821) [inline app.js:810]: Why is Rocketeer set up as a tank that has exploding bullets?

**Comment by haykam821** [comment]
The corresponding client pull request: https://github.com/CommunityDiep/client/pull/2

File: app.js (hunk 1 above)
Comment (Legoburner73) [reply app.js:1]: That was causing issues on my end. Blame my harddrive. Too scared to update NODE cos low storage.

File: app.js (hunk 2 above)
Comment (Legoburner73) [reply app.js:663]: dev cheats. it was set to 'false' always.

File: app.js (hunk 3 above)
Comment (Legoburner73) [reply app.js:810]: That was from another test.

**Pull request merged**